src/wrappers/OsipiBase.py

```diff
@@ -16,6 +16,10 @@ def __init__(self, bvalues=None, thresholds=None, bounds=None, initial_guess=Non
         self.initial_guess = np.asarray(initial_guess) if initial_guess is not None else None
         self.use_bounds = True
         self.use_initial_guess = True
+
+        # Validate the inputs
+        self.osipi_validate_inputs()
```

Comment:
I think bvalues are not required here, and I believe that will fail the check. Should we require it at this level? I hate to add too many requirements so perhaps we need 2 levels of checking. A lenient and a strict where the lenient passes `None` but strict does not. Then the check here can be lenient but the check before fitting can be strict.